scripts/install_narrative_docker.sh

```diff
@@ -110,5 +114,6 @@ jupyter nbextension install $(pwd)/codeCell --symlink --sys-prefix
 jupyter nbextension enable codeCell/main --sys-prefix
 
 jupyter nbextension enable --py --sys-prefix widgetsnbextension
+jupyter nbextension enable --py --sys-prefix clustergrammer_widget
```

Comment:
It might make sense to replace this logic with a call to `nbextention/install.sh`